span/bode.py

```diff
@@ -20,18 +20,18 @@ def __post_init__(self):
         self.timeArray = np.linspace(1/self.samplerate, self.voltageOut.size/self.samplerate, self.voltageOut.size)
     
     @staticmethod
-    def integral(x, y):
+    def integral(x: np.ndarray, y: np.ndarray) -> float:
```

Comment:
Depending on the shape of `x` and `y`, this method can also return an `np.ndarray`, so probably best to hint the output as either `float | np.ndarray` or `Union[float, np.ndarray]`, depending on Python version